src/components/Utbetaling/Detail/index.tsx

```diff
@@ -363,9 +362,7 @@ const UtbetalingerSammendrag = ({ utbetalinger, periode }: { utbetalinger: Utbet
                         Skriv ut
                     </Button>
                 </HStack>
-                <AriaNotification
-                    beskjed={`Det finnes ${utbetalinger.length} utbetalinger for valgt periode og filtrering`}
-                />
```

Comment:
Denne har timeout på seg, men gjorde det om til vanleg tekst pga at det alltid kan vera synleg. Det kan vera fint for vanlege folk også, ikkje berre dei med synsvanskar.